app/features/build_runner/build_runner.rb

```diff
@@ -283,8 +283,14 @@ def start
       else
         # No work queue? Just call the block then
         logger.debug("Not using a workqueue for build runner #{self.class}, this is probably a bug")
-        work_block.call
-        post_run_block.call
+
+        begin
+          work_block.call
+        rescue StandardError => ex
+          logger.error(ex)
+        ensure
+          post_run_block.call
```

Comment:
this is already called in an ensure block if we're using a workqueue to execute it

Reply:
Should this GitHub comment be an actual code comment?

Reply:
Good point